examples/basics/visuals/colorbar_visual.py

```diff
@@ -81,7 +83,7 @@ def get_mandlebrot_escape_values(width, height):
     return v_get_num_escape_turns(*grid).astype(np.float)
 
 
-def get_vertical_bar():
+def get_vertical_bar(pos, halfdim):
```

Comment:
I think we might want to use `size` instead of `halfdim`, it should be more intuitive for users. The first element is the typically short axis, the second is the typically long axis. For a vertical colorbar, this corresponds to `(w, h)`. Make sense?